packages/perseus/src/widgets/__tests__/radio/choice-icon_test.jsx

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+import {render, screen} from "@testing-library/react";
+import * as React from "react";
+
+import "@testing-library/jest-dom"; // Imports custom mathers
+
+import ChoiceIcon from "../../radio/choice-icon.jsx";
+
+const SAT_ICON_SIZE = 25;
+const LIBRARY_ICON_SIZE = 24;
+
+function renderChoiceIcon(options) {
+    const defaultOptions = {
+        pos: 0,
+        checked: false,
+        crossedOut: false,
+        focused: false,
+        hovered: false,
+        pressed: false,
+        correct: false,
+        showCorrectness: false,
+        product: "library",
+        multipleSelect: false,
+        reviewMode: false,
+        previouslyAnswered: false,
+        transparentBackground: false,
+    };
+
+    const overwrittenOptions = {...defaultOptions, ...options};
+
+    return render(<ChoiceIcon {...overwrittenOptions} />);
+}
+
+describe("choice icon", () => {
+    describe.each([["sat"], ["library"]])("product: %s", (product) => {
+        describe.each([[true], [false]])(
+            "multipleSelect: %s",
+            (multipleSelect) => {
+                it("renders with the correct border radius", () => {
+                    let expectedRadius;
+                    if (multipleSelect) {
+                        expectedRadius = 3;
+                    } else if (product === "sat") {
+                        expectedRadius = SAT_ICON_SIZE;
+                    } else {
+                        expectedRadius = LIBRARY_ICON_SIZE;
+                    }
+
+                    renderChoiceIcon({product, multipleSelect});
+
+                    const choiceWrapper = screen.getByTestId(
+                        `choice-icon__${product}-choice-icon`,
+                    );
+
+                    expect(choiceWrapper).toHaveStyle(
+                        `border-radius: ${expectedRadius}px`,
```

Comment:
**radio/check:** I have an action point to investigate why we went from toggling between check/radio to just using check. I think if we decide it was an accidental regression and we want to add that back it'll be outside the scope of this PR.

**testing CSS:** I guess I could remove this test and just bank on snapshot testing? I personally don't have qualms unit testing CSS, but you have more experience than me so I'm probably missing something.

Reply:
👍  i think Tamara was looking into the ARIA for checkbox/radio but let's create a separate ticket for that one.

We can see if we can get some of the native context back and re-think the accessibility text generator holistically rather than just going after this one solution.

Reply:
https://khanacademy.atlassian.net/browse/LP-12684